### ISSUE TITLE
rewrite is not working

### ISSUE THREAD

**Issue by k-miyake** [Azure/static-web-apps-cli#165]
**Describe the bug**
I have configured the routes in `staticwebapp.config.json`, but rewrite is not working as expected.
The config file is below. The application is Vue, created with Vite.

```
{
  "routes": [
    {
      "route": "/login",
      "rewrite": "/.auth/login/github"
    }
  ],
  "navigationFallback": {
    "rewrite": "/index.html",
    "exclude": [
      "/assets/*.{js,map}",
      "/assets/*.{png,jpg,gif}",
      "/assets/*"
    ]
  }
}
```

**To Reproduce**

1. Go to 'http://localhost:4280/login'
2. Provider-unidentified login pages are displayed.

**Expected behavior**
`/.auth/login/github` will be displayed

**Screenshots**
![swa-issue](https://user-images.githubusercontent.com/1104403/113801181-48111d80-9793-11eb-8df5-88456c83b88c.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser chrome
 - Version 89.0.4389.114

**Additional context**
The version of the CLI you are using is `0.3.0`.

**Comment by manekinekko** [comment]
Thank you for report this issue @k-miyake 👍 we'll review and merge #167 to fix it.

**Comment by k-miyake** [comment]
@manekinekko Thanks for the quick response. Looking forward to the fix.

**Comment by manekinekko** [comment]
@k-miyake this issue should be fixed now in `main` branch (#0934043)

**Comment by k-miyake** [comment]
@manekinekko Confirmed that it's fixed in `0.3.1-alpha`! Thanks.
